src/layer.ts

```diff
@@ -48,6 +52,13 @@ export const runtimeLookup: { [key: string]: RuntimeType } = {
   "python3.7": RuntimeType.PYTHON,
   "python3.8": RuntimeType.PYTHON,
   "python3.9": RuntimeType.PYTHON,
+  "dotnetcore3.1": RuntimeType.DOTNET,
+  dotnet6: RuntimeType.DOTNET,
+  java11: RuntimeType.JAVA,
+  "java8.al2": RuntimeType.JAVA,
+  "provided.al2": RuntimeType.CUSTOM,
```

Comment:
I was looking in the AWS docs for the list of supported runtimes for Lambda Extensions and I couldn’t find it. Instead I now see: `All Lambda runtimes support extensions.` https://docs.aws.amazon.com/lambda/latest/dg/using-extensions.html

This is probably partially due to the end of support for Node 10, Python 2.7, and Ruby 2.5. But it also means they added support for `python3.6`, `java8` , `go1.x`, and `provided`, which weren’t previously supported.

Maybe we can verify this and support them here as well?